music21/tablature.py

```diff
@@ -244,6 +244,14 @@ def getPitches(self):
 
         return pitchList
 
+    @staticmethod
```

Comment:
This feels like a module-level function to me.

Reply:
I agree, I moved it as a module-level function (I initially put it into a class because I didn't see any other module function in `tablature.py`.